[Bug]: RTS build error due to import wrong exported AST function

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Tsc Alias was not working as there was a wrong import of function which was exported from AST shared module which resulted in the below error 

> appsmith    | backend stdout | [2022-09-19 05:45:20,111]  - Multiple Spring Data modules found, entering strict repository configuration mode!
appsmith    | backend stdout | [2022-09-19 05:45:20,112]  - Bootstrapping Spring Data Reactive MongoDB repositories in DEFAULT mode.
appsmith    | backend stdout |
appsmith    | rts stdout | internal/modules/cjs/loader.js:905
appsmith    | rts stdout |   throw err;
appsmith    | rts stdout |   ^
appsmith    | rts stdout |
appsmith    | rts stdout | Error: Cannot find module '@controllers/socket'
appsmith    | rts stdout | Require stack:
appsmith    | rts stdout | - /opt/appsmith/rts/sockets/mongo.js
appsmith    | rts stdout | - /opt/appsmith/rts/sockets/index.js
appsmith    | rts stdout | - /opt/appsmith/rts/server.js
appsmith    | rts stdout |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
appsmith    | rts stdout |     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
appsmith    | rts stdout |     at Module.require (internal/modules/cjs/loader.js:974:19)
appsmith    | rts stdout |     at require (internal/modules/cjs/helpers.js:101:18)
appsmith    | rts stdout |     at Object.<anonymous> (/opt/appsmith/rts/sockets/mongo.js:18:18)
appsmith    | rts stdout |     at Module._compile (internal/modules/cjs/loader.js:1085:14)
appsmith    | rts stdout |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
appsmith    | rts stdout |     at Module.load (internal/modules/cjs/loader.js:950:32)
appsmith    | rts stdout |     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
appsmith    | rts stdout |     at Module.require (internal/modules/cjs/loader.js:974:19) {
appsmith    | rts stdout |   code: 'MODULE_NOT_FOUND',
appsmith    | rts stdout |   requireStack: [
appsmith    | rts stdout |     '/opt/appsmith/rts/sockets/mongo.js',
appsmith    | rts stdout |     '/opt/appsmith/rts/sockets/index.js',
appsmith    | rts stdout |     '/opt/appsmith/rts/server.js'
appsmith    | rts stdout |   ]
appsmith    | rts stdout | }
appsmith    | rts stdout |

### Steps To Reproduce

1. Try building RTS server
2. You will see the error on the console

### Public Sample App

_No response_

### Version

Both

--- a/app/rts/build.sh
+++ b/app/rts/build.sh
@@ -8,4 +8,4 @@ npx tsc && npx tsc-alias
 # Copying node_modules directory into dist as rts server requires node_modules to run server build properly. 
 # This was previously being done in dockerfile which was copying the symlinks to image rather than the whole directory of shared modules (e.g. AST)
 # Also, we copy node_modules with -L flag in order to follow the symlinks for @shared folder and copy the contents instead of just the symlink
-cp -rL node_modules ./dist
+cp -rL node_modules ./dist || cp -r node_modules ./dist

--- a/app/rts/build.sh
+++ b/app/rts/build.sh
@@ -8,4 +8,5 @@ npx tsc && npx tsc-alias
 # Copying node_modules directory into dist as rts server requires node_modules to run server build properly. 
 # This was previously being done in dockerfile which was copying the symlinks to image rather than the whole directory of shared modules (e.g. AST)
 # Also, we copy node_modules with -L flag in order to follow the symlinks for @shared folder and copy the contents instead of just the symlink
+# || command is added because on Mac -L flag doesn't work, so if the first command throws an error only then the command after || symbol will run
 cp -rL node_modules ./dist || cp -r node_modules ./dist

--- a/app/rts/build.sh
+++ b/app/rts/build.sh
@@ -8,5 +8,4 @@ npx tsc && npx tsc-alias
 # Copying node_modules directory into dist as rts server requires node_modules to run server build properly. 
 # This was previously being done in dockerfile which was copying the symlinks to image rather than the whole directory of shared modules (e.g. AST)
 # Also, we copy node_modules with -L flag in order to follow the symlinks for @shared folder and copy the contents instead of just the symlink
-# || command is added because on Mac -L flag doesn't work, so if the first command throws an error only then the command after || symbol will run
-cp -rL node_modules ./dist || cp -r node_modules ./dist
+cp -RL node_modules ./dist

--- a/app/rts/src/services/AstService.ts
+++ b/app/rts/src/services/AstService.ts
@@ -1,4 +1,4 @@
-import { extractIdentifiersFromCode } from "@shared/ast";
+import { extractInfoFromCode } from "@shared/ast";
 
 export default class AstService {
   static async getIdentifiersFromScript(
@@ -7,7 +7,7 @@ export default class AstService {
   ): Promise<any> {
     return new Promise((resolve, reject) => {
       try {
-        const extractions = extractIdentifiersFromCode(
+        const extractions = extractInfoFromCode(
           script,
           evalVersion
         );

